.github/workflows/wheels.yml

```
@@ -122,6 +123,65 @@ jobs:
 #          echo $(gcc --version)
 #        if: ${{ runner.os == 'Windows' && env.IS_32_BIT == 'true' }}
 
+      - name: Setup macOS
+        if: matrix.buildplat[0] == 'macos-11' || matrix.buildplat[0] == 'macos-14'
```

Maybe instead more future compatible?
```suggestion
        if: startswith(matrix.buildplat[0], 'macos')
```